bundles/core/org.eclipse.smarthome.core.thing.xml.test/src/test/resources/test-bundle-pool/ChannelTypesTest.bundle/ESH-INF/thing/thing-types.xml

```diff
@@ -26,6 +26,7 @@
 	
 	<channel-group-type id="channelgroup">
 		<label>Channel group</label>
+		<category>TEMPERATURE</category>
```

Comment:
Human readable "Temperature" as it is defined in https://www.eclipse.org/smarthome/documentation/development/bindings/thing-definition.html#channel-categories!